src/disco/containers/DiscoPane.js

```diff
@@ -176,8 +184,9 @@ export function loadedAddons(state) {
   );
 }
 
-export function mapStateToProps(state) {
+export function mapStateToProps(state, ownProps) {
   return {
+    platform: ownProps.params.platform,
```

Comment:
I dont see the `params` prop in the `PropTypes`.

Reply:
Ah, I put `platform` in there because it's what `mapStateToProps` returns. Should I add `params` as well? That's confusing ^_^

Reply:
Yeah, `params` are provided by react-router (like `location`). We need to declare all props, not only the mapped ones (that are redux only).

Reply:
Ah okay, cool.